auth/auth/driver/driver.py

```diff
@@ -164,7 +170,7 @@ async def create(self, username):
 
     async def _delete(self, gsa_email):
         try:
-            await self.iam_client.delete(f'/serviceAccounts/{gsa_email}/keys')
+            await self.iam_client.delete(f'/serviceAccounts/{gsa_email}')
```

Comment:
I can move this out into a separate PR but noticed that deletion of GSAs was broken and this fixes it.